src/AutomaticMechanismAnalysis.jl

```diff
@@ -273,11 +273,11 @@ function analyzespc(sim,spcname,t;N=10,tol=1e-3,branchthreshold=0.9,
     end
     push!(clusternames,"Stables")
 
-    @views dSdtspc = dSdt[length(sim.domain.phase.species)+1:end]
+    @views dSdtrxn = dSdt[length(sim.domain.phase.species)+1:end]
```

Comment:
Can you squash this commit to the relevant commit?

Reply:
This is a renaming of a variable that already existed on main so it doesn't squash in a readable way.